packages/neo-one-node-core/src/index.ts

```diff
@@ -18,10 +18,12 @@ export * from './Node';
 export * from './ScriptContainer';
 export * from './Serializable';
 export * from './Settings';
+export * from './Signer';
 export * from './Storage';
 export * from './StorageFlags';
 export * from './StorageItem';
 export * from './TrimmedBlock';
+export * from './TransactionData';
 export * from './Validator';
 export * from './ValidatorsCount';
 export * from './Witness';
```

Comment:
can remove handled in `csharp-core`

Reply:
actually, still need to add the `export * from ./TransactionData` to the `csharp-core` index.ts

Reply:
Done. `Signer` is already exported in `csharp-core` now.